paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormData.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.paymentsheet.forms
+
+class FormData(
+    val source: Map<String, Any?>,
+    val elementKeys: Map<String, String?>
```

Comment:
Can you add comments to explain what `source` and `elementKeys` are?

Reply:
yeah, variables are renamed, some refactored into functions.